openslides/agenda/models.py

```diff
@@ -269,7 +271,7 @@ def get_list_of_speakers(self, old_speakers_count=None, coming_speakers_count=No
                 speaker_dict['last_in_group'] = True
             list_of_speakers.append(speaker_dict)
 
-        # Parse actual speaker
+        # Parse current speaker
```

Comment:
Please add a new line `# TODO: Rename actual_speaker to current_speaker` or just do it right now if you like.